### PR TITLE
Create `PrintNumbersFrom1ToN.java`

### LINKED ISSUE #5215: Write a Java program to print numbers from 1 to n
### Description

Write a Java program to print numbers from 1 to n

```
Input  : 5
Output : 1 2 3 4 5
```
<details>
<summary>How to contribute</summary>

- Save the solution in `PrintNumbersFrom1ToN.java` file
- Add `PrintNumbersFrom1ToN.java` file in `program/print-numbers-from-1-to-n` folder

</details>

### PR DESCRIPTION
**Pull request by dineshlalam15**
Closes codinasion/codinasion#3870 
I've written a simple and understandable code to print 1 to n numbers.

### CODE REVIEW

**Comment by dineshlalam15** [comment]
Check out my new changes to the code here.
04d9d0eaf3175ddb5bb9086af45dbcc16376e0cd

**Review by anandfresh — CHANGES_REQUESTED** [review]
@dineshlalam15  assign yourself to this issue before raising a PR 
https://github.com/codinasion/program/issues/5215

File: program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+       Scanner sc = new Scanner(System.in);
+       System.out.print("Enter the number: ");
+       int n = sc.nextInt();
+       print(n);
+  }
+  static void print(int n){
+    for(int i = 1; i <=n; i++){
+      System.out.print(i + " ");
+  }
+}
```
Comment (anandfresh) [inline program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java:12]: class closing bracket is missing .
Can you do an unit testing before submitting the PR for a review

**Review by anandfresh — CHANGES_REQUESTED** [review]

File: program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java (hunk 1 above)
Comment (dineshlalam15) [reply program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java:13]: I really apologise for the inconvenience caused. I won't let it happen next time :)

**Review by joao-vitor-souza — APPROVED** [review]

File: program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java (hunk 1 above)
Comment (joao-vitor-souza) [reply program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java:15]: > I really apologise for the inconvenience caused. I won't let it happen next time :)

No need to apologize, we're here to help you.

**Review by anandfresh — APPROVED** [review]

**Pull request merged**